src/pages/workspaces/workspace/JobHistory.js

```diff
@@ -153,19 +154,36 @@ class JobHistoryContent extends Component {
           },
           columns: [
             {
-              headerRenderer: () => h(HeaderCell, ['Workflow']),
+              headerRenderer: () => h(HeaderCell, ['Job']),
               cellRenderer: ({ rowIndex }) => {
-                const { methodConfigurationNamespace, methodConfigurationName, submitter } = submissions[rowIndex]
-                return div({}, [
-                  div({}, [
-                    methodConfigurationNamespace !== namespace && span({ style: styles.deemphasized }, [
-                      `${methodConfigurationNamespace}/`
+                const { methodConfigurationNamespace, methodConfigurationName, submitter, submissionId } = submissions[rowIndex]
+                return h(Fragment, [
+                  div([
```

Comment:
Do we not want to leave the empty props for consistency? I'm fine with not, I just thought at some point we had said we would.

Reply:
I think it was `children` always being an array that I was pushing for, because hyperscript is particular about types.